ci-operator/config/redhat-appstudio/rhtap-installer/redhat-appstudio-rhtap-installer-main.yaml

```diff
@@ -91,6 +91,11 @@ tests:
     yamllint -c "$CONFIG_FILE" .
   container:
     from: rhtap-installer
+- as: rhtap-dump-external-resources
```

Comment:
why we need this?

Reply:
We will use Sprayproxy forwards Github Event requests to the ephemeral clusters. So it needs to clean up the invalid Pac servers from Sprayproxy server.